lib/services/network_helper.dart

```diff
@@ -1,17 +1,14 @@
-import 'dart:convert';
-
 import 'package:http/http.dart' as http;
 
 class NetworkHelper {
   NetworkHelper(this.url);
   final String url;
 
-  Future<dynamic> getData() async {
+  Future<String> getData() async {
     http.Response response = await http.get(url);
 
     if (response.statusCode == 200) {
-      String data = response.body;
-      return jsonDecode(data);
+      return response.body;
     } else {
       print(response.statusCode);
       return null;
```

Comment:
Maybe you could find a way to return decoded JSON from response.body immediately then use null-aware operator for every call to this method.